src/Microsoft.TemplateEngine.Core/Expressions/MSBuild/MSBuildStyleEvaluatorDefinition.cs

```diff
@@ -254,6 +103,61 @@ private static ITokenTrie GetSymbols(IProcessorState processor)
             return tokens;
         }
 
+        private static void ConfigureConverters(ITypeConverter obj)
```

Comment:
The static methods ConfigureConverters() and TryHexConvert() are copied in multiple implementations of SharedEvaluatorDefinition. Seems like we only need 1 copy - perhaps in SharedEvaluatorDefinition itself?